dev/Repeater/TestUI/RepeaterTestUIPage.xaml

```diff
@@ -41,8 +41,8 @@
                 <TextBlock>MinColumnSpacing</TextBlock>
                 <TextBox x:Name="lineSpacing" Text="10"></TextBox>
                 <TextBlock>LineAlignment</TextBlock>
-                <TextBlock>(Near/Middle/Far/SpaceEqually)</TextBlock>
-                <TextBox x:Name="lineAlingment" Text="Near"></TextBox>
+                <TextBlock>(Start/Center/End/SpaceAround/SpaceBetween/SpaceEvenly)</TextBlock>
```

Comment:
Is this related to the rest of this change? From the description it sounded like it was only about fixing the downlevel and OS build stuff.

Reply:
Not related but this was causing a crash in the test app due to a rename that was missed in a previous change.